start.py

```diff
@@ -590,8 +621,16 @@ def _find_frozen_openpype(use_version: str = None,
 
     if not is_inside:
         # install latest version to user data dir
-        version_path = bootstrap.install_version(
-            openpype_version, force=True)
+        if not os.getenv("OPENPYPE_HEADLESS_MODE"):
```

Comment:
I quite don't like checking only existence of value as it may be set to `"0"`, "`foo`",... All of that would result in `True`.

Reply:
But that is quite standard process - you check for existence of the variable as it has only boolean type value - if it exists, it's True no matter to what it it set.